### ISSUE TITLE
`generateName` attribute is not processed correctly and workspace is failing to start

### ISSUE THREAD

**Issue by ibuziuk** [eclipse/che#21810]
### Describe the bug

`generateName` attribute is not processed correctly and workspace is failing to start

```
Failed to create a new workspace from the devfile, reason: Failed to create a new workspace. Unable to create devworkspace: Either DevWorkspace `metadata.name` or `metadata.generateName` is required.
```

### Che version

7.52

### Steps to reproduce

Reproducible against https://workspaces.openshift.com/#https://github.com/che-incubator/quarkus-api-example

### Expected behavior

workspace is started without problems

### Runtime

OpenShift

### Installation method

OperatorHub

### Environment

Amazon

**Comment by ibuziuk** [comment]
@olexii4 @akurinnoy we need to make sure that the issue is fixed in 7.56 for Dev Spaces 3.2

**Comment by akurinnoy** [comment]
Closing this issue as it's not reproducible in 7.56.
